test/widgets/test_countdown.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2015 Tycho Andersen
```

Comment:
Looks like some copypasta errors here, don't give me credit for your work :)

Reply:
Haha. Dammit. Copied from a previous one, amended it, but obviously forgot to copy again!

No doubt you'll have other comments so I'll add this to my "to fix" list for the commit.

Reply:
You're kidding? I was sure I got all of them...